pkg/reconciler/namespace/namespace.go

```diff
@@ -439,3 +448,41 @@ func (c *Controller) enqueueNamespaces(ctx context.Context, selector labels.Sele
 
 	return nil
 }
+
+type clusterEnqueueStrategy int
+
+const (
+	enqueueScheduled clusterEnqueueStrategy = iota
+	enqueueUnscheduled
+	enqueueNothing
+)
+
+// enqueueStrategyForCluster determines what namespace enqueueing strategy
+// should be used in response to a given cluster state. Also returns a boolean
+// indication of whether to enqueue the cluster in the future to respond to a
+// impending cordon event.
+func enqueueStrategyForCluster(cl *clusterv1alpha1.Cluster) (clusterEnqueueStrategy, bool) {
```

Comment:
nit: would name the return values when there is a bool like that. Makes it easier to read.

Reply:
done